.circleci/config.yml

```diff
@@ -279,7 +279,7 @@ jobs:
     - *restore_cache
     - *yarn_install
 
-    - run: ./scripts/install-angular-snapshots.sh
+    - run: node ./scripts/install-angular-snapshots.js
```

Comment:
Could you do this with just `cat package.json | egrep -o '@angular/[^"]+' ` piping to `yarn install --force`?

Reply:
Problem is that we need to map to `github:angular/$PACKAGE_NAME-builds` and AFAIK, you cannot only output a specific match group with grep?

Reply:
I can change the script to just run `yarn install URLS --force` though. That sounds like it would make the script a bit simpler. Thoughts?

Reply:
@jelbourn I've simplified this a bit. Let me know if you have something simpler in mind.

Reply:
I had in mind basically this:
```sh
cat package.json | egrep -o '@angular/[^"]+' | sed 's|@|github:|' | sed 's|.*|&-builds|' | xargs yarn add --force
```

Reply:
Nice. I didn't think of just replacing the `@` with the desired prefix. Made the changes!